source/serializables/error.js

```diff
@@ -1,3 +1,11 @@
 // Make errors serializable
 Space.Error.mixin(Space.messaging.Ejsonable);
 Space.Error.type('Space.Error');
+
+// Make errors versionable
+Space.Error.mixin(Space.messaging.Versionable);
```

Comment:
Yes, i just found out – it is a bug in `space:base`, will push a fix PR in a few minutes!